### PR TITLE
Switch eng demos to Acrablocks

### PR DESCRIPTION
**Pull request by ZhmakaAS**
Switch demos to acrablock by default. Updated config and image version to 0.91.0.
Set up `--poison_detect_enable=false` flag for all demos.

### CODE REVIEW

**Review by Lagovas — APPROVED** [review]
great job

File: django-transparent/docker-compose.django-transparent.yml [hunk 1]
```diff
@@ -4,9 +4,9 @@ services:
     # Creating keys:
     # - ./.acrakeys/acra-server/${ACRA_CLIENT_ID}_storage
```
Comment (vixentael) [inline django-transparent/docker-compose.django-transparent.yml:5]: we don't create asyn keys anymore, do we?

File: django-transparent/docker-compose.django-transparent.yml [hunk 2]
```diff
@@ -17,8 +17,8 @@ services:
             --client_id=""
             --tls_cert=/ssl/acra-client.crt
             --generate_acrawriter_keys
```
Comment (vixentael) [inline django-transparent/docker-compose.django-transparent.yml:19]: should we?

File: timescaledb/docker-compose.timescaledb.yml [hunk 3]
```diff
@@ -17,6 +17,7 @@ services:
             --client_id=""
             --tls_cert=/ssl/acra-client.crt
             --generate_acrawriter_keys
+            --generate_symmetric_storage_key
```
Comment (vixentael) [inline timescaledb/docker-compose.timescaledb.yml:20]: if we are using acrablocks, why generating asym keys as well?

**Review by vixentael — CHANGES_REQUESTED** [review]

File: django-transparent/docker-compose.django-transparent.yml (hunk 1 above)
Comment (vixentael) [reply django-transparent/docker-compose.django-transparent.yml:5]: so that's a workaround, understood!

**Review by vixentael — APPROVED** [review]

**Pull request merged**